spec/base_spec.rb

```diff
@@ -226,8 +226,6 @@ def execute_with_failure
 
   context 'when an action class declares no `returns` or `fails_with`' do
     before do
-      stub_const('ApplicationAction', Class.new(ActiveActions::Base))
-      stub_const('User', Class.new(ActiveRecord::Base))
```

Comment:
(unrelated change; these stubbed constants are already stubbed in an outer `describe` block; doing so again here is redundant/unnecessary.)